component/task.go

```diff
@@ -22,3 +22,9 @@ type TaskLaunchInfo struct {
 	// arguments to that program.
 	Arguments []string
 }
+
+// TaskResult is the result value that must be returned by the RunTask
+// function.
+type TaskResult struct {
+	ExitCode int
+}
```

Comment:
While we're here, is there an easy way to place the spawned Task ids from the plugin outside of the Tasks plugin proto result into its own variable? Not on this struct `TaskResult`, but probably somewhere here inside `component/task`? If it's a bigger change, feel free to ignore for now.